core/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation group: 'org.jdbi', name: 'jdbi3-sqlobject', version: '3.34.0'
     implementation group: 'org.jdbi', name: 'jdbi3-postgres', version: '3.34.0'
     implementation 'org.springframework.boot:spring-boot-starter-data-jdbc:2.7.10'
+    implementation 'org.springframework.boot:spring-boot-starter-webflux:2.7.10'
```

Comment:
This is for Spring's `WebClient`, which may be a bit more fully featured than what we need, but since we're using Spring already, this is what we get out-of-the-box.